contribs/gmf/examples/style/mobileapp-main.less

```diff
@@ -0,0 +1,23 @@
+@import (inline) '../../../../node_modules/openlayers/css/ol.css';
+@import (inline) 'reset.css';
+
+@import 'mobileapp-vars.less';
+
+@import '../../../../node_modules/more-or-less/less/css3/_transition';
+@import '../../../../node_modules/more-or-less/less/css3/_transform';
```

Comment:
Instead of relying on `more-or-less`, we should use `bootstrap`'s mixins.
Same for `reset.css`.

I'll give it a try.